alodataset/coco_base_dataset.py

```diff
@@ -341,7 +355,12 @@ def __call__(self, image, target):
 
 
 if __name__ == "__main__":
-    coco_dataset = CocoBaseDataset(sample=True)
+    coco_dataset = CocoBaseDataset(sample=False, img_folder="test2017")
+    #checking if regular getitem works
+    stuff=coco_dataset.getitem(0)
```

Comment:
you can simply `stuff = coco_dataset[0]`

Reply:
I actually commited this file by mistake, it was already merged in #247